### ISSUE TITLE
Add a fallback font for when FA is disabled

### ISSUE THREAD

**Issue by SaschaMann** [exercism/exercism#5718]
The icons for formatting in the mentoring editor don't seem to have a fallback font. It's common practice to disable third-party fonts and/or google fonts as part of fingerprinting and tracking protection, so I imagine this will be encountered somewhat frequently.

![grafik](https://user-images.githubusercontent.com/20866761/131988187-ef4325c7-f7a5-4c12-b264-24d0aa1564e1.png)

**Comment by iHiD** [comment]
Just as an FYI, this is all part of the library, so fixing it will be quite a lot of work.

**Comment by SaschaMann** [comment]
I'd be fine with not having that bar at all in that case. It's a bit weird to have invisible buttons that are even highlighted sometimes.

**Comment by iHiD** [comment]
> I'd be fine with not having that bar at all in that case. 

Yeah - again, it would the challenge though. We'd have to hack the external library, and I'm not sure that font-detection is something that you can do easily. The app doesn't know whether your call to Google Fonts succeeds or not.

**Comment by iHiD** [comment]
(or in this case font-awesome)

**Comment by SleeplessByte** [comment]
> The app doesn't know whether your call to Google Fonts succeeds or not.

`FontFaceObserver` can help us here. See: https://fontfaceobserver.com/
